spec/spec_helper.rb

```diff
@@ -1,5 +1,6 @@
 require 'capybara/rspec'
 require 'rails_helper'
+require 'support/factory_girl'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.